code/datums/gamemodes/campaign/loadout_items/_TGMC/head.dm

```diff
@@ -32,41 +32,82 @@
 
 
 /datum/loadout_item/helmet/standard
-	name = "M10X helmet"
-	desc = "A standard TGMC combat helmet. Apply to head for best results."
-	item_typepath = /obj/item/clothing/head/modular/m10x
-	jobs_supported = list(SQUAD_MARINE, SQUAD_SMARTGUNNER)
+	name = "L Helmet"
+	desc = "A standard TDF combat helmet. Apply to head for best results."
+	req_desc = "Requires a light armor suit."
+	item_typepath = /obj/item/clothing/head/modular/tdf
+	jobs_supported = list(SQUAD_MARINE)
 	loadout_item_flags = LOADOUT_ITEM_ROUNDSTART_OPTION|LOADOUT_ITEM_DEFAULT_CHOICE
+	item_whitelist = list(
+		/obj/item/clothing/suit/modular/tdf/light/shield = ITEM_SLOT_OCLOTHING,
+	)
+
+/datum/loadout_item/helmet/medium
+	name = "M Helmet"
+	desc = "A standard TDF combat helmet. Apply to head for best results."
+	req_desc = "Requires a medium armor suit."
+	item_typepath = /obj/item/clothing/head/modular/tdf/medium
+	jobs_supported = list(SQUAD_MARINE)
+	loadout_item_flags = LOADOUT_ITEM_ROUNDSTART_OPTION
+	item_whitelist = list(
+		/obj/item/clothing/suit/modular/tdf/shield = ITEM_SLOT_OCLOTHING,
+	)
+
+/datum/loadout_item/helmet/heavy
+	name = "H Helmet"
+	desc = "A standard TDF combat helmet. Apply to head for best results."
+	req_desc = "Requires a heavy armor suit."
+	item_typepath = /obj/item/clothing/head/modular/tdf/heavy
+	jobs_supported = list(SQUAD_MARINE)
+	loadout_item_flags = LOADOUT_ITEM_ROUNDSTART_OPTION
+	item_whitelist = list(
+		/obj/item/clothing/suit/modular/tdf/heavy/shield = ITEM_SLOT_OCLOTHING,
```

Comment:
the white/blacklists are type sensitive, so you need to make sure it specifically includes all valid options - i.e. overclocked variants etc.

I assume this issue will be here for many of the whitelists.